agenta-web/cypress/e2e/regex-evaluation.cy.ts

```diff
@@ -0,0 +1,148 @@
+import {app_id} from "../support/commands/utils"
+
+describe("Regex Evaluation workflow", () => {
+    context("When creating variants and testsets", () => {
+        it("Should successfully create app variant and testset", () => {
+            cy.createVariantsAndTestsets()
+        })
+    })
+
+    context("When navigating to Evaluation Page", () => {
+        it("Should reach the Evaluation Page", () => {
+            cy.visit(`/apps/${app_id}/playground`)
+            cy.clickLinkAndWait('[data-cy="app-evaluations-link"]')
+            cy.url().should("include", "/evaluations")
+        })
+    })
+
+    context("When no Variant and Testset are Selected", () => {
+        beforeEach(() => {
+            cy.visit(`/apps/${app_id}/playground`)
+            cy.clickLinkAndWait('[data-cy="app-evaluations-link"]')
+            cy.url().should("include", "/evaluations")
+        })
+
+        it("Should display a warning to select Variant", () => {
+            cy.clickLinkAndWait('[data-cy="regex-button"]')
+            cy.clickLinkAndWait('[data-cy="start-new-evaluation-button"]')
+            cy.get(".ant-message-notice-content")
+                .should("contain.text", "Please select a variant")
+                .should("exist")
+            cy.wait(3000)
```

Comment:
why do we need wait here?

Reply:
Optional but to track the notification
After 3 seconds notification should not exist